### ISSUE TITLE
Dask_avro cant run with dask 1.17.0

### ISSUE THREAD

**Issue by nirizr** [rmax/dask-avro#3]
See following snippet:

```
(venv) user@machine:~/dir$ ipython
Python 3.5.2 (default, Nov 23 2017, 16:37:01) 
Type 'copyright', 'credits' or 'license' for more information
IPython 6.2.1 -- An enhanced Interactive Python. Type '?' for help.

In [1]: import dask

In [2]: dask.__version__
Out[2]: '0.17.0'

In [3]: import dask_avro
---------------------------------------------------------------------------
ImportError                               Traceback (most recent call last)
<ipython-input-3-ea91e8dcfef9> in <module>()
----> 1 import dask_avro

~/dir/venv/lib/python3.5/site-packages/dask_avro/__init__.py in <module>()
     13 from dask import delayed
     14 from dask.bytes import read_bytes
---> 15 from dask.bytes.core import OpenFileCreator
     16 
     17 

ImportError: cannot import name 'OpenFileCreator'

In [4]: 

```

**Comment by rmax** [comment]
Thank you for the report. Will see if it's a quick fix.

**Comment by nirizr** [comment]
See commit https://github.com/dask/dask/commit/b7be7d8007ed7c2b425b5998b85dde60d82052c5 where `open_files` replaces `OpenFileCreator`.

**Comment by rmax** [comment]
Fixed in https://github.com/rmax/dask-avro/commit/faf81785accbca4d086fefbba472cfd11f72e34b and released [v0.1.2](https://pypi.org/project/dask-avro/0.1.2/)

**Comment by nirizr** [comment]
Thanks, I can verify this is working for me.
